libs/pages/elewa/home/src/lib/components/home-jobs-section/home-jobs-section.component.html

```diff
@@ -1,8 +1,8 @@
-<section class="home-jobs-section">
-    <span >
-        Become part of the family
-    </span>
-    <span >
-        View <a class="homeLink" routerLink="/jobs">open opportunities.</a>
-    </span>
-</section>
\ No newline at end of file
+
+ <elewa-group-social-impact-call-to-action-section
+ [message1]="message1"
+ [message2]="message2"
+ [linkText]="linkText"
+ [link]="link"
+ ></elewa-group-social-impact-call-to-action-section>
```

Comment:
This should be a reusable component in a shared lib like features/components/banners